shared/src/metabase/mbql/util.cljc

```diff
@@ -281,6 +283,22 @@
     [(op :guard temporal-extract-ops) field & args]
     [:temporal-extract field (temporal-extract-ops->unit [op (first args)])]))
 
+(defn- desugar-divide-with-extra-args [expression]
```

Comment:
Honey SQL 1 is happy to compile divide with > 2 args, e.g. `[:/ 1 2 3]` => `1 / 2 / 3`. Honey SQL 2 explicitly disallows this.

To make everyone's lives easier I fixed this generally by converting `[:/ 1 2 3]` to `[:/ [:/ 1 2] 3]` and so forth, which is actually what it means anyway. Hopefully this will mean non-SQL drivers don't have to work around this either in the future